order.go

```diff
@@ -91,12 +91,6 @@ type OrderUpdateShippingParams struct {
 	TrackingNumber *string `form:"tracking_number"`
 }
 
-// OrderReturnParams is the set of parameters that can be used when returning orders.
-type OrderReturnParams struct {
-	Params `form:"*"`
-	Items  []*OrderItemParams `form:"items"`
-}
-
```

Comment:
Why was this removed?

Reply:
It's in the same package, different file FWIW

Reply:
It's fine to move it to a different file in the same package (the user can't even tell).